src/core/CoreStore/index.ts

```diff
@@ -75,14 +80,20 @@ export default class CoreStore {
 
     this.setView = this.setView.bind(this);
 
+    this.setPage = this.setPage.bind(this);
+
     this.setSection = this.setSection.bind(this);
   }
 
-  setView(pathname: string): void {
-    this.view = getViewFromPathname(pathname);
+  setView(view: CoreView | PathwaysView): void {
```

Comment:
Super minor readability nit. All of the other places where we combine pathways and core types in this file have pathways first.
```suggestion
  setView(view: PathwaysView | CoreView): void {
```